packages/adblocker/src/resources.ts

```diff
@@ -314,7 +314,7 @@ export default class Resources {
     const scriptlet = this.getRawScriptlet(name);
```

Comment:
imao this previous approach was quite fine

```suggestion
    const scriptlet = this.getRawScriptlet(name) || this.getResourceAsScriptlet(name);
```